packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/actions.ts

```diff
@@ -644,3 +644,6 @@ export const fetchSDDLimitsSuccess = (
     sddLimits
   }
 })
+export const updateSddTransactionFinished = () => ({
+  type: AT.UPDATE_SDD_TRANSACTION_FINISHED
+})
```

Comment:
Maybe you could take the new boolean value here.

Reply:
yeah, I was thinking in first place to make this super simple, but we can make it dynamic ;)